buildpipeline/DotNet-CoreClr-Trusted-Mac.json

```diff
@@ -1,115 +1,163 @@
 {
-  "build": [
```

Comment:
why is this so radically different? Looks broken.

Reply:
The whitespace formatting got updated.  @JohnTortugo can you revert the whitespace only changes or put them in a separate commit?